\documentclass{article}
\input{settings}

\begin{document}

\section{Results and Discussion}
\subsection{Growth rate estimation using linear regression}
% For the sake of simplicity, we assumed that there is no copy number aberrations or rearrangements in the cancer genome. Furthermore, we also assumed that every single nucleotide variant (SNV) is unique to each subtype and not inherited by its descendants. Under these conditions, we simulated sequencing reads under some birth time parameters (Figure 3). Given the number of reference and variant reads in each SNV locus, which subtype is responsible for that mutation, and the abundance ratio of each subtype, we could estimate the birth time parameters of all subtypes. Pseudo time parameters could be estimated precisely if there is a sufficient number of SNV loci (Figure 4).

First, I conducted the double check of the neutral tumor evolution which is identified in the previous work~\cite{williams2016identification}.
In their study, if neutral evolution theory applies to passenger mutation, the number of passenger mutations per allelic frequency $M(f)$ follows $1/f$ power law,
\begin{equation}
 M(f) = \mu_e \left( \frac{1}{f} - \frac{1}{f_{max}}\right),\label{neutral_evolution}
\end{equation}
where $\mu_e$ denotes the mutation rate per effective cell division.
And they set $R^2 \leq 0.98$ as the stringent criteria to decide whether each tumor sample obeys neutral evolution or not.
If we consider that we cannot estimate the mutation rate and the birth time of a subtype at the same time as long as only a single time point data is available
because there is a trade-off between the mutation rate and the birth time, the inverse of $\mu_e$ has a information about the birth time of the subtype.
Thus, if a tumor sample obeys neutral evolution, we can estimate the birth time and growth rate of the tumor sample using linear regression, which is conducted as \figref{TB_linear_regression}.

\begin{figure}[H]
 \begin{minipage}[c]{0.47\hsize}
 \begin{center}
  \includegraphics[bb=3 4 479 489, width=0.9\hsize]{../../mutect/mutect/TB_nextera_3/TB.call_stats.extract.keep.tumor_f.sorted.bold.pdf}
 \end{center}
    \caption{
  Variant allelic frequency distribution of the sample TB~\cite{williams2016identification}.
  Mutations with higher frequency ($>0.4$) and lower frequency ($<0.4$) correspond to driver mutations and passenger mutations respectively.
  }\label{TB_vaf}
\end{minipage}
 \hspace{1truecm}
\begin{minipage}[c]{0.47\hsize}
 \begin{center}
 \includegraphics[bb=3 28 460 471, width=0.9\hsize]{../../mutect/mutect/TB_nextera_3/TB.call_stats.extract.keep.tumor_f.sorted.filtered.reverse.bold.pdf}
 \end{center}
   \caption{
  The cumulative number of mutations $M(f)$ plotted against the inverse allelic frequency $1/f$
  using the sample TB~\cite{williams2016identification}.
  The red line represents the result of the linear regression.
 Though $R^2 = 0.976 < 0.98$ deviates from the stringent criteria~\cite{williams2016identification}, this tumor almost obeys the neutral evolution.
  }
 \label{TB_linear_regression}
\end{minipage}
\end{figure}

However, if there are multiple subtypes, $M(f)$ generally does not follow $1/f$ power law as the previous work showed using simulated data~\cite{williams2016identification}.
We conducted double check of this result using our simulated reads. As a result, our simulation also showed deviation from the $1/f$ power law.

\begin{figure}[H]
 \begin{minipage}[c]{0.47\hsize}
 \begin{center}
  \includegraphics[bb=4 18 467 481,width=0.9\hsize]{../src/read_generation/generated/subtype2_topology0_snv_diploid/vaf_coverage100/0.5/0.3/5/coverage100_snv1000_seed1.reads.1.vaf_coverage100_snv1000_seed1.reads.2.vaf.pdf}
 \end{center}
    \caption{
  Variant allelic frequency distribution of the simulated reads ($t_1 = 0.5, t_2=0.15, n_1=0.5, n_2=0.5, N=10^6, \beta=0.5, \text{\#snvs}=1000$).
  Variant allelic frequency distribution has multiple peaks if there are two subtypes in the tumor.
  }\label{simulated_vaf}
\end{minipage}
 \hspace{1truecm}
\begin{minipage}[c]{0.47\hsize}
 \begin{center}
  \includegraphics[bb=3 28 487 471,width=0.9\hsize]{../src/read_generation/generated/subtype2_topology0_snv_diploid/neutral_evolution_plot/0.5/0.3/5/coverage100_snv1000_seed1.reads.vaf.filtered.reverse.pdf}
 \end{center}
   \caption{
  The cumulative number of mutations $M(f)$ plotted against the inverse allelic frequency $1/f$
  using the simulated reads ($t_1 = 0.5, t_2=0.15, n_1=0.5, n_2=0.5, \text{\#snvs}=1000$).
  The red line represents the result of the linear regression.
 The tumor which consists of multiple subtypes cause deviation from the $1/f$ power law though each subtype follows neutral evolution.
  }
 \label{simulated_linear_regression}
\end{minipage}
\end{figure}

Therefore, if a tumor sample consists of multiple subtypes, we cannot estimate the growth rate of each subtype using linear regression.
I resolved this problem using probabilistic modeling mentioned in the method section.

\subsection{Relationship between the birth time and the variant allele frequency distribution}
VAF distribution of subtype $i$, $p(x_i | t_i, n_i)$ can be calculated given the birth time $t_i$ and the abundance ratio $n_i$ using equation \eqref{vaf_dist_norm} mentioned in the method section. To investigate the time ($t_i$) dependency and the drift strength ($\beta_i$) dependency of the VAF distribution, I showed the violin plots against these variables (\figref{fig: vaf_against_time}, \figref{vaf_against_strength}). From these plots, we can see that the VAF is more likely to be fixed ($x_i = 1$) in the tumor cell population if the subtype arose earlier. Also, the VAF is more likely to be fixed ($x_i = 1$) in the tumor cell population if the effect of the genetic drift is stronger.

Using this relationship conversely, we can estimate the birth time of each subtype from the observed VAF distribution.
However, the observed VAF frequency is a mixture of various subtypes as the \figref{simulated_vaf}.
Thus, we must conduct the subtype decomposition using mixture modeling mentioned in the method section.

\begin{figure}[H]
 \begin{minipage}[c]{0.47\hsize}
 \begin{center}
  \includegraphics[bb=5 15 669 581,width=0.9\hsize]{../../dc1/figures/drift_time_all_annotated_shrinked2_jp.pdf}
 \end{center}
    \caption{
  VAF distribution $p(x_i | t_i, n_i)$ plotted against pseudo time $t_i$ ($n_i=0.1, N=10^6, \beta_i=0.5$).
  We can see that the earlier the subtype arose, the higher VAF fixation probability is.
  }
    \label{fig: vaf_against_time}
\end{minipage}
 \hspace{1truecm}
\begin{minipage}[c]{0.47\hsize}
 \begin{center}
\includegraphics[bb=28 2 476 663,width=0.5\hsize]{../../cge/dc_plot/drift_strength.pdf}
 \end{center}
   \caption{
 VAF distribution $p(x_i | t_i, n_i)$ plotted against the drift strength $\beta_i$ ($t_i= n_i=0.1, N=10^6$).
 We can see that the earlier the subtype arose, the higher VAF fixation probability is.
  }
   \label{vaf_against_strength}
\end{minipage}
\end{figure}

\subsection{Estimation of the birth time and the abundance ratio of each subtype using binomial distribution}
Using the parameter estimation modeling represented as equation \ref{param_estimation_binomial}, I conducted the estimation of the birth time $t_i$ and the abundance ratio $n_i$ of each subtype.

First, I started with solving more easier problem than presented in the method section (equation \ref{binom_estimation_model}).
I assumed that we know the subtype $i_k$ in which each SNV $k$ is originated and the inheritance pattern to its descendants $h_k$ along side with the number of variant and total reads ($m, M$).
In this case, the probability that we get the read count along with additional information $(m,M,i,h) = \left\{(m_1, M_1, i_1, h_1), \cdots, (m_K, M_K, i_K, h_K) \right\}$ as a result of NGS sequencing and the variant calling is,
\begin{equation}
 p(m_{1:K}|M_{1:K}, i_{1:K}, h_{1:K}, n_{1:I}, t_{1:I}) = \prod_{k=1}^{K} \sum_{x_{i_k}} p(x_{i_k} | i_k, h_k, n_{i_k}, t_{i_k}, t_{\mathrm{child}(i_k)}) p (m_k | M_k, n, x).
\end{equation}

Using this marginal probability, I estimated the birth time $t$ and abundance ratio $n$ using Fletcher-Reeves conjugate gradient algorithm~\cite{fletcher1964function} according to the following maximum likelihood estimation,
\begin{equation}
 (t_{1:I}, n_{1:I}) = \argmax_{t_{1:I}, n_{1:I}} \ln p(m_{1:K} | M_{1:K}, i_{1:K}, h_{1:K}, n_{1:I}, t_{1:I}).
\end{equation}

Simulated data was generated using the method mentioned in the method section.% write!
For each number of SNVs, sequencing data was generated 10 times independently. And for each simulated data, maximum likelihood estimation was conducted 10 times using randomly selected initial parameters $t, n$. The parameters with the least Root-mean-square deviation (RMSD) was selected as the best estimate for that simulated data.
RMSD is calculated as $\mathrm{RMSD}_t = \sqrt{\sum_{i=1}^{I} (t_{i}^{(est)} - t_{i}^{(true)})^2 / I}, \mathrm{RMSD}_n = \sqrt{\sum_{i=1}^{I} (n_{i}^{(est)} - n_{i}^{(true)})^2 / I}$, where the superscript $est$ means the best estimate and $true$ means the parameter which is used to generate simulated reads. Normal cell population birth time is defined as $t_{0}=1$, and the abundance ratio $n_{0}$ is given as 0.1.

\begin{table}[H]
\begin{center}
 \caption{True parameters which is used to generate sequencing data}
 \label{tab: true_parameters_4_2_original}
  \begin{tabular}[t]{lcccc}\toprule
  Population type & Tumor subtype 1 & Tumor subtype 2 & Tumor subtype 3 & Tumor subtype 4\\\midrule
  Birth time & 9.000460e-01 & 8.100828e-01 & 6.481035e-01 & 7.291118e-01\\
  Abundance ratio & 1.000000e-01 & 2.000000e-01 & 4.000000e-01 & 2.000000e-01\\\bottomrule
 \end{tabular}
\end{center}
\end{table}

$N = 10^6, \beta = 0.5$. % write!

\begin{figure}[H]
 \subfigure[Accuracy of the birth time estimation]{\includegraphics[bb=3 15 475 446,width=0.45\hsize]{../src/alpha_n_inherited/accuracy_result_seed_poisson/4_2_original_total1000000_td1000_cell1000000.rmsd.all.t.snv.pdf}}
 \subfigure[Accuracy of the abundance ratio estimation]{\includegraphics[bb=3 15 475 446,width=0.45\hsize]{../src/alpha_n_inherited/accuracy_result_seed_poisson/4_2_original_total1000000_td1000_cell1000000.rmsd.all.n.snv.pdf}}
   \caption{
 Accuracy of the birth time (t) and abundance ratio (n) estimation against the number of SNVs. Outliers are detected by means of deviation from the $1.5 \times IQR$, where $IQR$ is the interquartile range. Whiskers show the maximum and minimum estimate except for the outliers. We can see that if there are sufficient number of SNVs, we can accurately estimate the birth time and the growth rate as long as $(i,h)$ are given.
  }
 \label{fig: binom_subtype_inheritance}
\end{figure}

The results of the parameter estimation shows that we can infer the birth time and growth rate accurately if there are thousands of SNVs, which corresponds to the number of mutations detected in the whole genome sequencing (\figref{fig: binom_subtype_inheritance}).
In reality, however, we cannot tell the subtype in which each SNV is originated and the inheritance pattern to its descendants only from the sequencing reads.
To move on to the next step, I investigated whether we can remove the latter assumption by marginalizing unknown inheritance patterns with equal probability, which is formulated as follows,

\begin{gather}
 p(m_{1:K}|M_{1:K}, i_{1:K}, n_{1:I}, t_{1:I}) = \prod_{k=1}^{K} \sum_{h_k=0}^{H_k-1} \frac{1}{H_{i_k}} \sum_{x_{i_k}} p(x_{i_k} | i_k, h_k, n_{i_k}, t_{i_k}, t_{\mathrm{child}(i_k)}) p (m_k | M_k, n, x),\\
 (t_{1:I}, n_{1:I}) = \argmax_{t_{1:I}, n_{1:I}} \ln p(m_{1:K} | M_{1:K}, i_{1:K}, n_{1:I}, t_{1:I}),
\end{gather}
where I put $p(h_k | i_k) = \frac{1}{H_{i_k}}$ in the \eqref{binom_estimation_model}.
True parameters of $t$ and $n$ and the simulated reads are the same as that of mentioned above (\tabref{table: true_parameters_4_2_original}).
The result of the parater estimation shows that

\end{document}