\documentclass{article}
\input{settings}

\begin{document}

\section{Results and Discussion}
\subsection{Birth time estimation using linear regression}
% For the sake of simplicity, we assumed that there is no copy number aberrations or rearrangements in the cancer genome. Furthermore, we also assumed that every single nucleotide variant (SNV) is unique to each subtype and not inherited by its descendants. Under these conditions, we simulated sequencing reads under some birth time parameters (Figure 3). Given the number of reference and variant reads in each SNV locus, which subtype is responsible for that mutation, and the abundance ratio of each subtype, we could estimate the birth time parameters of all subtypes. Pseudo time parameters could be estimated precisely if there is a sufficient number of SNV loci (Figure 4).

First, I conducted the double check of the neutral tumor evolution which is identified in the previous work~\cite{williams2016identification}.
In their study, ``if neutral evolution theory applies to passenger mutation, the number of passenger mutations per allelic frequency $M(f)$ follows $1/f$ power law,
\begin{equation}
 M(f) = \mu_e \left( \frac{1}{f} - \frac{1}{f_{max}}\right),\label{neutral_evolution}
\end{equation}
where $\mu_e$ denotes the mutation rate per effective cell division.
And they set $R^2 \leq 0.98$ as the stringent criteria to decide whether each tumor sample obeys neutral evolution or not.''
If we consider that we cannot estimate the mutation rate and the birth time of a subtype at the same time as long as only a single time point data is available
because there is a trade-off between the mutation rate and the birth time, the inverse of $\mu_e$ has a information about the birth time of the subtype.
Thus, ``if a tumor sample obeys neutral evolution, we can estimate the birth time of the tumor sample using linear regression'', which is conducted as \figref{TB_vaf}, \ref{TB_linear_regression}.

\begin{figure}[H]
 \begin{minipage}[c]{0.47\hsize}
 \begin{center}
  \includegraphics[bb=3 4 479 489, width=0.9\hsize]{../../mutect/mutect/TB_nextera_3/TB.call_stats.extract.keep.tumor_f.sorted.bold.pdf}
 \end{center}
    \caption{
  Variant allelic frequency distribution of the colorectal tumor sample TB~\cite{williams2016identification}.
  Mutations with higher frequency and lower frequency each correspond to driver mutations and passenger mutations respectively.
  }\label{TB_vaf}
\end{minipage}
 \hspace{1truecm}
\begin{minipage}[c]{0.47\hsize}
 \begin{center}
 \includegraphics[bb=3 28 460 471, width=0.9\hsize]{../../mutect/mutect/TB_nextera_3/TB.call_stats.extract.keep.tumor_f.sorted.filtered.reverse.bold.pdf}
 \end{center}
   \caption{
  The cumulative number of mutations $M(f)$ plotted against the inverse allelic frequency $1/f$
  using the sample TB~\cite{williams2016identification}.
  The red line represents the result of the linear regression.
 Though $R^2 = 0.976 < 0.98$ deviates from the stringent criteria~\cite{williams2016identification}, this tumor almost obeys the neutral evolution.
  }
 \label{TB_linear_regression}
\end{minipage}
\end{figure}

However, ``if there are multiple subtypes, $M(f)$ generally does not follow $1/f$ power law as the previous work showed using simulated data''~\cite{williams2016identification}.
We conducted double check of their result using our simulated reads using Algorithm \ref{algo: read_simulation}. As a result, our simulation also showed deviation from the $1/f$ power law~\figref{simulated_vaf}, \ref{simulated_linear_regression}.

\begin{figure}[H]
 \begin{minipage}[c]{0.47\hsize}
 \begin{center}
  \includegraphics[bb=4 18 467 481,width=0.9\hsize]{./coverage100_snv1000_seed1.reads.1.vaf_coverage100_snv1000_seed1.reads.2.vaf.pdf}
 \end{center}
    \caption{
  Variant allelic frequency distribution of the simulated reads ($t_1 = 0.5, t_2=0.15, n_1=0.5, n_2=0.5, N=10^6, \beta=0.5, \text{\#SNVs}=1000$).
  }\label{simulated_vaf}
\end{minipage}
 \hspace{1truecm}
\begin{minipage}[c]{0.47\hsize}
 \begin{center}
  \includegraphics[bb=3 28 487 471,width=0.9\hsize]{../src/read_generation/generated/subtype2_topology0_snv_diploid/neutral_evolution_plot/0.5/0.3/5/coverage100_snv1000_seed1.reads.vaf.filtered.reverse.pdf}
 \end{center}
   \caption{
  The cumulative number of mutations $M(f)$ plotted against the inverse allelic frequency $1/f$
  using the simulated reads ($t_1 = 0.5, t_2=0.15, n_1=0.5, n_2=0.5, \text{\#SNVs}=1000$).
  The red line represents the result of the linear regression.
 The tumor which consists of multiple subtypes cause deviation from the $1/f$ power law though each subtype follows neutral evolution.
  }
 \label{simulated_linear_regression}
\end{minipage}
\end{figure}

Therefore, if a tumor sample consists of multiple subtypes, we cannot estimate the birth time of each subtype using linear regression.
I resolved this problem using probabilistic modeling mentioned in the method section.

Moreover, there was a case that the tumor consists of two subtypes though the $M(f)$ and $1/f$ have the linear relationship in our simulated data (\figref{VAF_distribution_generated}, \ref{inverse_VAF_cumulative_generated}).

\begin{figure}[H]
 % \includegraphics[bb=1 15 467 481,width=0.3\hsize]{../src/read_generation/generated/subtype2_topology0_snv_forvaf/vaf_coverage100/0.5/0.4/3/coverage100_snv1000_seed1.reads.1.vaf.pdf}
 % \includegraphics[bb=1 15 467 481,width=0.3\hsize]{../src/read_generation/generated/subtype2_topology0_snv_forvaf/vaf_coverage100/0.5/0.4/3/coverage100_snv1000_seed1.reads.2.vaf.pdf}
 % \includegraphics[bb=1 15 467 481,width=0.3\hsize]{../src/read_generation/generated/subtype2_topology0_snv_forvaf/vaf_coverage100/0.5/0.4/3/coverage100_snv1000_seed1.reads.1.vaf_coverage100_snv1000_seed1.reads.2.vaf.pdf}
 \includegraphics[bb=3 28 461 471,width=0.3\hsize]{../src/read_generation/generated/subtype2_topology0_snv_forvaf/coverage100_snv1000_seed1.reads.1.vaf.pdf}
 \includegraphics[bb=3 28 461 471,width=0.3\hsize]{../src/read_generation/generated/subtype2_topology0_snv_forvaf/coverage100_snv1000_seed1.reads.2.vaf.pdf}
 \includegraphics[bb=3 28 461 471,width=0.3\hsize]{../src/read_generation/generated/subtype2_topology0_snv_forvaf/coverage100_snv1000_seed1.reads.1.vaf_coverage100_snv1000_seed1.reads.2.vaf.pdf}
 \caption{The VAF distribution of the sequencing data which is generated by our probabilistic modeling. NGS read counts are simulated using the birth time $t_1=0.5, t_2=0.2$, and abundance ratio $n_1=0.3,  n_2=0.7$. Higher allelic frequency peaks seen in subtype 1 and 2 correspond to the fixed variant alleles. Only the VAF distribution of subtype 1+2 can be observed in the bulk sequencing.}
 \label{VAF_distribution_generated}
\end{figure}

\begin{figure}[H]
 % \includegraphics[bb=3 28 487 471,width=0.3\hsize]{../src/read_generation/generated/subtype2_topology0_snv_forvaf/neutral_evolution_plot/0.5/0.4/3/coverage100_snv1000_seed1.reads.1.vaf.filtered_0.05_0.2.reverse.pdf}
 % \includegraphics[bb=3 28 487 471,width=0.3\hsize]{../src/read_generation/generated/subtype2_topology0_snv_forvaf/neutral_evolution_plot/0.5/0.4/3/coverage100_snv1000_seed1.reads.2.vaf.filtered_0.1_0.5.reverse.pdf}
 % \includegraphics[bb=3 28 487 471,width=0.3\hsize]{../src/read_generation/generated/subtype2_topology0_snv_forvaf/neutral_evolution_plot/0.5/0.4/3/coverage100_snv1000_seed1.reads.vaf.filtered_0.12_0.24_with_grad.reverse.pdf}
 \includegraphics[bb=3 28 460 471,width=0.3\hsize]{../src/read_generation/generated/subtype2_topology0_snv_forvaf/coverage100_snv1000_seed1.reads.1.vaf.filtered_0.05_0.2.reverse.pdf}
 \includegraphics[bb=3 28 460 471,width=0.3\hsize]{../src/read_generation/generated/subtype2_topology0_snv_forvaf/coverage100_snv1000_seed1.reads.2.vaf.filtered_0.1_0.5.reverse.pdf}
 \includegraphics[bb=3 28 460 471,width=0.3\hsize]{../src/read_generation/generated/subtype2_topology0_snv_forvaf/coverage100_snv1000_seed1.reads.vaf.filtered_0.12_0.24.reverse.pdf}
 \caption{The cumulative number of mutations $M(f)$ plotted against the inverse allelic frequency $1/f$ using the sequencing data which is generated by our probabilistic modeling (birth time $t_1=0.5, t_2=0.2$, and abundance ratio $n_1=0.3,  n_2=0.7$). If we exclude fixation region which can be seen in \figref{VAF_distribution_generated}, subtype 1 and 2 obeys neutral evolution. (I plotetd $M(f)$ against $0.05 < f < 0.2$ for subtype 1, and $0.1 < f < 0.5$ for subtype 2.) This is a concordant result with the previous study~\cite{williams2016identification}. Subtype 1 + 2 also shows linear relationsip between $M(f)$ and $1/f$ with $\mu_e = 32.523$.  However, in reality, this is a mixture of two subtypes having different mutation rates per effective cell division ($\mu_e = 9.081$ and $\mu_e=18.230$). This suggests that the linear regression can cause wrong subtype structure estimation.}
 \label{inverse_VAF_cumulative_generated}
\end{figure}

When we consider the SNVs originated in subtype 1 and those originated in subtype 2 separately, each subtypes follows the neutral evolution.
This shows that our probabilistic modeling of the VAF distribution is concordant with that of the previous work~\cite{williams2016identification} if the fixation and loss region in the VAF disrtibution is excluded. In the previous work, the effect of the genetic drift is not considered, thus the fixation and loss probability is not considered. Thus our Wright-Fisher modeling includes the modeling of the previous study.

However when we consider these SNVs altogether, the estimate of $\mu_e$ differed from that of each subtype. This suggests that the linear regression can cause wrong subtype structure and birth time estimation.

\subsection{Relationship between the birth time and the variant allele frequency distribution}
VAF distribution of subtype $i$, $p(x_i | t_i, n_i)$ can be calculated given the birth time $t_i$ and the abundance ratio $n_i$ using equation \eqref{vaf_dist_norm} mentioned in the method section. To investigate the time ($t_i$) dependency and the drift strength ($\beta_i$) dependency of the VAF distribution, I showed the violin plots against these variables (\figref{fig: vaf_against_time}, \ref{vaf_against_strength}). From these plots, we can see that the VAF is more likely to be fixed ($x_i = 1$) in the tumor cell population if the subtype arose earlier. Also, the VAF is more likely to be fixed ($x_i = 1$) in the tumor cell population if the effect of the genetic drift is stronger.

Using this relationship conversely, we can estimate the birth time of each subtype from the observed VAF distribution.
However, the observable VAF frequency distribution is a mixture of various subtypes as the figure ``Subtype 1 + 2'' shown in \figref{VAF_distribution_generated}.
Thus, we must conduct the subtype decomposition using mixture modeling described in the method section.

\begin{figure}[H]
 \begin{minipage}[c]{0.47\hsize}
 \begin{center}
  \includegraphics[bb=5 15 669 581,width=0.9\hsize]{../../dc1/figures/drift_time_all_annotated_shrinked2_jp.pdf}
 \end{center}
    \caption{
  VAF distribution $p(x_i | t_i, n_i)$ plotted against pseudo time $t_i$ ($n_i=0.1, N=10^6, \beta_i=0.5$).
  We can see that the earlier the subtype arose, the higher the VAF fixation probability is.
  }
    \label{fig: vaf_against_time}
\end{minipage}
 \hspace{1truecm}
\begin{minipage}[c]{0.47\hsize}
 \begin{center}
\includegraphics[bb=28 2 476 663,width=0.5\hsize]{../../cge/dc_plot/drift_strength.pdf}
 \end{center}
   \caption{
 VAF distribution $p(x_i | t_i, n_i)$ plotted against the drift strength $\beta_i$ ($t_i= n_i=0.1, N=10^6$).
 We can see that the earlier the subtype arose, the higher the VAF fixation probability is.
  }
   \label{vaf_against_strength}
\end{minipage}
\end{figure}

\subsection{Estimation of the birth time and the abundance ratio of each subtype using binomial distribution}
Using the parameter estimation modeling represented as equation \eqref{binom_estimation_model}, I conducted the estimation of the birth time $t_i$ and the abundance ratio $n_i$ of each subtype.

% First, I started with solving more easier problem than presented in the method section (equation \ref{binom_estimation_model}).
% I assumed that we know the subtype $i_k$ in which each SNV $k$ is originated and the inheritance pattern to its descendants $h_k$ along side with the number of variant and total reads ($m, M$).
% In this case, the probability that we get the read count along with additional information $(m,M,i,h) = \left\{(m_1, M_1, i_1, h_1), \cdots, (m_K, M_K, i_K, h_K) \right\}$ as a result of NGS sequencing and the variant calling is,
% \begin{equation}
%  p(m_{1:K}|M_{1:K}, i_{1:K}, h_{1:K}, n_{1:I}, t_{1:I}) = \prod_{k=1}^{K} \sum_{x_{i_k}} p(x_{i_k} | i_k, h_k, n_{i_k}, t_{i_k}, t_{\mathrm{child}(i_k)}) p (m_k | M_k, n, x).
% \end{equation}

% Using this marginal probability, I estimated the birth time $t$ and abundance ratio $n$ using Fletcher-Reeves conjugate gradient algorithm~\cite{fletcher1964function} according to the following maximum likelihood estimation,
% \begin{equation}
%  (t_{1:I}, n_{1:I}) = \argmax_{t_{1:I}, n_{1:I}} \ln p(m_{1:K} | M_{1:K}, i_{1:K}, h_{1:K}, n_{1:I}, t_{1:I}).
% \end{equation}

% Simulated data was generated using the method mentioned in the method section.% write!
% For each number of SNVs, sequencing data was generated 10 times independently. And for each simulated data, maximum likelihood estimation was conducted 10 times using randomly selected initial parameters $t, n$. The parameters with the least Root-mean-square deviation (RMSD) was selected as the best estimate for that simulated data.
% RMSD is calculated as $\mathrm{RMSD}_t = \sqrt{\sum_{i=1}^{I} (t_{i}^{(est)} - t_{i}^{(true)})^2 / I}, \mathrm{RMSD}_n = \sqrt{\sum_{i=1}^{I} (n_{i}^{(est)} - n_{i}^{(true)})^2 / I}$, where the superscript $est$ means the best estimate and $true$ means the parameter which is used to generate simulated reads. Normal cell population birth time is defined as $t_{0}=1$, and the abundance ratio $n_{0}$ is given as 0.1.

% \begin{table}[H]
% \begin{center}
%  \caption{True parameters which is used to generate sequencing data}
%  \label{tab: true_parameters_4_2_original}
%   \begin{tabular}[t]{lcccc}\toprule
%   Population type & Tumor subtype 1 & Tumor subtype 2 & Tumor subtype 3 & Tumor subtype 4\\\midrule
%   Birth time & 9.000460e-01 & 8.100828e-01 & 6.481035e-01 & 7.291118e-01\\
%   Abundance ratio & 1.000000e-01 & 2.000000e-01 & 4.000000e-01 & 2.000000e-01\\\bottomrule
%  \end{tabular}
% \end{center}
% \end{table}

% $N = 10^6, \beta = 0.5$. % write!

% \begin{figure}[H]
%  \subfigure[Accuracy of the birth time estimation]{\includegraphics[bb=3 15 475 446,width=0.45\hsize]{../src/alpha_n_inherited/accuracy_result_seed_poisson/4_2_original_total1000000_td1000_cell1000000.rmsd.all.t.snv.pdf}}
%  \subfigure[Accuracy of the abundance ratio estimation]{\includegraphics[bb=3 15 475 446,width=0.45\hsize]{../src/alpha_n_inherited/accuracy_result_seed_poisson/4_2_original_total1000000_td1000_cell1000000.rmsd.all.n.snv.pdf}}
%    \caption{
%  Accuracy of the birth time (t) and abundance ratio (n) estimation against the number of SNVs. Outliers are detected by means of deviation from the $1.5 \times IQR$, where $IQR$ is the interquartile range. Whiskers show the maximum and minimum estimate except for the outliers. We can see that if there are sufficient number of SNVs, we can accurately estimate the birth time and the growth rate as long as $(i,h)$ are given.
%   }
%  \label{fig: binom_subtype_inheritance}
% \end{figure}

% The results of the parameter estimation shows that we can infer the birth time and growth rate accurately if there are thousands of SNVs, which corresponds to the number of mutations detected in the whole genome sequencing (\figref{fig: binom_subtype_inheritance}).
% In reality, however, we cannot tell the subtype in which each SNV is originated and the inheritance pattern to its descendants only from the sequencing reads.
% To move on to the next step, I investigated whether we can remove the latter assumption by marginalizing unknown inheritance patterns with equal probability, which is formulated as follows,



First, I started with solving more easier problem than that presented in the method section (equation \eqref{binom_estimation_model}).
I assumed that we know the subtype $i_k$ in which each SNV $k$ is originated as well as the number of variant and total reads ($m, M$).
In this case, the probability that we get the read count along with additional information $(m,M,i,h) = \left\{(m_1, M_1, i_1, h_1), \cdots, (m_K, M_K, i_K, h_K) \right\}$ as a result of NGS sequencing and the variant calling is,
\begin{equation}
  p(m_{1:K}|M_{1:K}, i_{1:K}, n_{1:I}, t_{1:I}) = \prod_{k=1}^{K} \sum_{h_k=0}^{H_k-1} \frac{1}{H_{i_k}} \sum_{x_{i_k}} p(x_{i_k} | i_k, h_k, n_{i_k}, t_{i_k}, t_{\mathrm{child}(i_k)}) p (m_k | M_k, n, x),
\end{equation}
where I put $p(h_k | i_k) = \frac{1}{H_{i_k}}$ in the equation \eqref{binom_estimation_model} to marginalize the unknown inheritance patterns with equal probabilities.
Using this marginal probability, I estimated the birth time $t$ and abundance ratio $n$ using Fletcher-Reeves conjugate gradient algorithm~\cite{fletcher1964function} according to the following maximum likelihood estimation,
\begin{equation}
 (t_{1:I}, n_{1:I}) = \argmax_{t_{1:I}, n_{1:I}} \ln p(m_{1:K} | M_{1:K}, i_{1:K}, n_{1:I}, t_{1:I})
\end{equation}

Simulated data was generated using Algorithm \ref{algo: read_simulation} represented in the method section.
For each number of SNVs, sequencing data was generated 10 times independently. And for each simulated data, maximum likelihood estimation was conducted 10 times using randomly selected initial parameters $t, n$. The parameters with the least Root-Mean-Square deviation (RMSD) was selected as the best estimate for that simulated data.
The birth time of the normal cell is defined as $t_{0}=1$, and the abundance ratio $n_{0} = 0.1$ is given.

\begin{table}[H]
\begin{center}
 \caption{True parameters which is used to generate sequencing data (\#Subtypes=4)}
 \label{tab: true_parameters_4_2_original}
  \begin{tabular}[t]{lcccc}\toprule
  Population type & Tumor subtype 1 & Tumor subtype 2 & Tumor subtype 3 & Tumor subtype 4\\\midrule
  Birth time & 9.000460e-01 & 8.100828e-01 & 6.481035e-01 & 7.291118e-01\\
  Abundance ratio & 1.000000e-01 & 2.000000e-01 & 4.000000e-01 & 2.000000e-01\\\bottomrule
 \end{tabular}
\end{center}
\end{table}

\begin{figure}[H]
 \subfigure[Accuracy of the birth time ($t$) estimation]{\includegraphics[bb=3 15 475 446,width=0.45\hsize]{../src/alpha_n_pattern_multinomial/accuracy_result_seed_poisson/4_2_original_total1000000_td1000_cell1000000.rmsd.all.t_snv.pdf}}
 \subfigure[Accuracy of the abundance ratio ($n$) estimation]{\includegraphics[bb=3 15 475 446,width=0.45\hsize]{../src/alpha_n_pattern_multinomial/accuracy_result_seed_poisson/4_2_original_total1000000_td1000_cell1000000.rmsd.all.n_snv.pdf}}
   \caption{
 Accuracy of the birth time ($t$) and abundance ratio ($n$) estimation against the number of SNVs. Outliers are detected by means of deviation from the $1.5 \times IQR$, where $IQR$ is the interquartile range. Whiskers show the maximum and minimum estimates except for the outliers. We can see that if there are sufficient number of SNVs, we can accurately estimate the birth time and the abundance ratio as long as the subtype in which each SNV is originated is given.
  }
 \label{fig: binom_subtype_given}
\end{figure}

The results of the parameter estimation shows that we can infer the birth time and abundance ratio accurately if there are larger number of SNVs (\figref{fig: binom_subtype_given}).
In reality, however, we cannot tell the subtype in which each SNV is originated only from the sequencing reads.
To move on to the next step, I investigated whether we can remove this unrealistic assumption by marginalizing the subtypes with equal probabilities, which is formulated as follows,
\begin{gather}
 p(m_{1:K}|M_{1:K}, n_{1:I}, t_{1:I}) = \prod_{k=1}^{K} \sum_{i_k=1}^{I} \frac{1}{I} \sum_{h_k=0}^{H_k-1} \frac{1}{H_{i_k}} \sum_{x_{i_k}} p(x_{i_k} | i_k, h_k, n_{i_k}, t_{i_k}, t_{\mathrm{child}(i_k)}) p (m_k | M_k, n, x),\label{eq: binom_subtype_ungiven}\\
 (t_{1:I}, n_{1:I}) = \argmax_{t_{1:I}, n_{1:I}} \ln p(m_{1:K} | M_{1:K}, n_{1:I}, t_{1:I}),
\end{gather}
where I put $p(i_k) = \frac{1}{I}$ and $p(h_k | i_k) = \frac{1}{H_{i_k}}$ in the equation \eqref{binom_estimation_model}.
True parameters of $t$ and $n$ and the simulated reads are the same as that of mentioned above (\tabref{tab: true_parameters_4_2_original}).
As a result of parameter estimation using equation \eqref{eq: binom_subtype_ungiven}, I could not estimate the true birth time and abundance ratio even if there are thousands of SNVs (Supplementary \figref{fig: binom_subtype_ungiven}).
This is because gradient descent estimates fall into the optima which are different from the true parameters.

Also, I tried to estimate the birth time and abundance ratio by adding only the maximum probability component with regard to the subtype in which each SNV is originated.
The formulation is given as follows.
\begin{gather}
 p(m_{1:K}|M_{1:K}, n_{1:I}, t_{1:I}) = \prod_{k=1}^{K} \max_{i_k=1}^{I} \left[\sum_{h_k=0}^{H_k-1} \frac{1}{H_{i_k}} \sum_{x_{i_k}} p(x_{i_k} | i_k, h_k, n_{i_k}, t_{i_k}, t_{\mathrm{child}(i_k)}) p (m_k | M_k, n, x)\right],\label{eq: binom_subtype_ungiven}\\
 (t_{1:I}, n_{1:I}) = \argmax_{t_{1:I}, n_{1:I}} \ln p(m_{1:K} | M_{1:K}, n_{1:I}, t_{1:I}),
\end{gather}
In this case too, the gradient descent estimates fall into the optima which are different from the true parameters, thus I could not infer the true birth time and abundance ratio even if there are thousands of SNVs (Supplementary \figref{fig: binom_subtype_ungiven_mle}).
These problems was essentially caused by the miss subtype-labeling of each SNV, that is, the subtype which gives the maximum probabilistic component does not match the true subtype.
Though we cannot expect the perfect subtype-labeling because we only have a single data with regard to each SNV, systematic labeling error cause the estimation failure.
In the next subsection, I conducted the estimation of the birth time and abundance ratio using the generative model (equation \eqref{read_emission_modeling}), which is concordant with the read simulation in this study, thus there is a guarantee that the maximum likelihood estimates converge to the true parameters given sufficient number of data.

\subsection{Estimation of the birth time and the abundance ratio of each subtype using NGS read emission probability}
Based on the EM algorithm derived in the method section (Algorithm \ref{EM_algorithm}), I conducted the maximum likelihood estimation of the birth time and abundance ratio of each subtype.
Simulated reads were generated with the true parameter shown in \tabref{tab: true_parameters_0.5_0.4_3}. Normal cell population birth time is defined as $t_{0}=1$, and the abundance ratio $n_{0} = 0$ is given.

\begin{table}[H]
\begin{center}
 \caption{True parameters which is used to generate sequencing data (\#Subtypes=2)}
 \label{tab: true_parameters_0.5_0.4_3}
  \begin{tabular}[t]{lcc}\toprule
  Population type & Tumor subtype 1 & Tumor subtype 2\\\midrule
   Birth time & 5.000000e-01 & 2.000000e-01\\
   Abundance ratio & 3.000000e-01 & 7.000000e-01\\\bottomrule
 \end{tabular}
\end{center}
\end{table}

The result of the birth time and abundance ratio estimation varying the number of SNVs from 1000 to 5000 while the sequencing depth is fixed to 100 is shown in (\figref{fig: read_emission_snv}).

\begin{figure}[H]
 % \subfigure[Accuracy of the birth time ($t$) estimation]{\includegraphics[bb=3 15 475 446,width=0.45\hsize]{../src/mutation_em/rmsd_qsub_snv_seed_rand_marged/rmsd.t.pdf}}
 % \subfigure[Accuracy of the abundance ratio ($n$) estimation]{\includegraphics[bb=3 15 475 446,width=0.45\hsize]{../src/mutation_em/rmsd_qsub_snv_seed_rand_marged/rmsd.n.pdf}}
 \subfigure[Accuracy of the birth time ($t$) estimation]{\includegraphics[bb=3 33 460 427,width=0.45\hsize]{../src/mutation_em/rmsd_qsub_snv_seed_rand/bests.t.rev.pdf}}
 \subfigure[Accuracy of the abundance ratio ($n$) estimation]{\includegraphics[bb=3 33 460 417,width=0.45\hsize]{../src/mutation_em/rmsd_qsub_snv_seed_rand/bests.n.rev.pdf}}
   \caption{
 Accuracy of the birth time ($t$) and abundance ratio ($n$) estimation against the number of SNVs. Outliers are detected by means of deviation from the $1.5 \times IQR$, where $IQR$ is the interquartile range. Whiskers show the maximum and minimum estimates except for the outliers. We can see that if there are sufficient number of SNVs, we can accurately estimate the birth time and abundance ratio even if we do not know the subtype in which each SNV is originated.
  }
 \label{fig: read_emission_snv}
\end{figure}

From this result, we can inferr the birth time and abundance ratio accurately with larger number of SNVs.
Though the birth time estimation requires greater number of SNVs, smaller number of SNVs is sufficient for the abundance ratio estimation.
I suppose this is because abundance ratio $n$ appears explicitly in the equation \eqref{read_emission_modeling} while the birth time $t$ affect the likelihood through VAF distribution.

Finally, I estimated the birth time and abundance ratio varying the sequencing depth from 10 to 100 while the number of SNVs is fixed to 3000 (\figref{fig: read_emission_coverage}).

\begin{figure}[H]
 % \subfigure[Accuracy of the birth time ($t$) estimation]{\includegraphics[bb=3 15 475 446,width=0.45\hsize]{../src/mutation_em/rmsd_qsub_coverage_seed_rand_marged/rmsd.t.pdf}}
 % \subfigure[Accuracy of the abundance ratio ($n$) estimation]{\includegraphics[bb=3 15 475 446,width=0.45\hsize]{../src/mutation_em/rmsd_qsub_coverage_seed_rand_marged/rmsd.n.pdf}}
 \subfigure[Accuracy of the birth time ($t$) estimation]{\includegraphics[bb=3 28 460 427,width=0.45\hsize]{../src/mutation_em/rmsd_qsub_coverage_seed_rand_param/0.5/0.4/3/3000/bests.t.rev.pdf}}
 \subfigure[Accuracy of the abundance ratio ($n$) estimation]{\includegraphics[bb=3 28 460 417,width=0.45\hsize]{../src/mutation_em/rmsd_qsub_coverage_seed_rand_param/0.5/0.4/3/3000/bests.n.rev.pdf}}
   \caption{
 Accuracy of the birth time ($t$) and abundance ratio ($n$) estimation against the sequencing depth. Outliers are detected by means of deviation from the $1.5 \times IQR$, where $IQR$ is the interquartile range. Whiskers show the maximum and minimum estimates except for the outliers. The birth time and the abundance ratio can be estimated more acculately with the higher sequencing depth.
  }
 \label{fig: read_emission_coverage}
\end{figure}

The birth time and abundance ratio were estimated more acculately with the higher sequencing depth.
% Though the accuracy of the abundance ratio estimation improved as the sequencing depth gets higher, the accuracy of the birth time estimation highly varied.
% To assess the effect of sequencing depth more accurately, I must conduct this experiment with larger number of SNVs and the larger number of replicates.

\end{document}